lib/flow/get-next-steps-for-user.js

```diff
@@ -77,6 +77,10 @@ module.exports = (task, profile) => {
         nextSteps.push(discardedByAsru);
       }
 
+      if (!taskIsOpen(task) && userIsAsruAdmin(profile)) {
```

Comment:
Does this mean we can recover a granted task as well? Do we specifically want to check for `rejected` here?

Reply:
you're right - it does in client but it also wants to be more specific here - I'll update